.github/workflows/localnet-test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: "get localnet stats"
         working-directory: ./e2e/monitor
         # XXX should this be a make command?
-        run: go test -timeout 30m -v .
+        run: go test -timeout 60m -v .
```

Comment:
frustratingly, what we save in build time we lose a bit in runtime due to the `FaultDisputeGame` way to do things vs `OutputOracle`